### PR TITLE
DOC-6537: Import notary cert

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/5.2/application-networks/creating/members/cpi.html#import-the-notary-certificate

### CODE REVIEW

File: content/en/platform/corda/5.1/application-networks/creating/members/cpi.md [hunk 1]
```diff
@@ -17,7 +17,8 @@ This section describes how to build a {{< tooltip >}}member{{< /tooltip >}} {{<
 2. [Generate the Group Policy File]({{< relref "#generate-the-group-policy-file" >}})
```
Comment (conalsmith-r3) [inline content/en/platform/corda/5.1/application-networks/creating/members/cpi.md:17]: we need to update 5.2 docs as well afaik

File: content/en/platform/corda/5.1/application-networks/creating/members/cpi.md [hunk 2]
```diff
@@ -198,6 +199,51 @@ Corda validates that uploaded CPIs are signed with a trusted key. To trust your
 Use an alias that will remain unique over time, taking into account that certificate expiry will require new certificates with the same X.500 name as existing certificates.
 {{< /note >}}
 
+## Import the Notary Certificate
```
Comment (conalsmith-r3) [inline content/en/platform/corda/5.1/application-networks/creating/members/cpi.md:202]: I think we need to do this before we create the CPI file

File: content/en/platform/corda/5.1/application-networks/creating/members/cpi.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.1/application-networks/creating/members/cpi.md:17]: updated 5.2

**Review by conalsmith-r3 — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**